centaur/src/main/scala/centaur/api/CentaurCromwellClient.scala

```diff
@@ -69,7 +70,8 @@ object CentaurCromwellClient {
     Try(Await.result(x(), timeout)) recoverWith {
       case _: TimeoutException |
            _: StreamTcpException |
-           _: IOException if !CromwellManager.isReady && attempt < 5 =>
+           _: IOException |
+           _: UnsupportedContentTypeException if !CromwellManager.isReady && attempt < 5 =>
```

Comment:
I imagine something changed in the cromwell client between when we ignored centaur tests (specifically restart ones) and now because now that I re-enabled restart tests, this is a new failure that appeared when trying to make a request to Cromwell while it's down. Apparently the response (whatever it is) is not json so the client fails to unmarshall with this exception.